rs/canister_client/read_state_response_parser/BUILD.bazel

```
@@ -0,0 +1,32 @@
+load("@rules_rust//rust:defs.bzl", "rust_library", "rust_test")
+
+package(default_visibility = ["//visibility:public"])
+
+DEPENDENCIES = [
+    # Keep sorted.
+    "//rs/canonical_state",
+    "//rs/certification",
+    "//rs/crypto/tree_hash",
+    "//rs/tree_deserializer",
+    "//rs/types/types",
+    "@crate_index//:serde",
+    "@crate_index//:serde_cbor",
+]
+
+DEV_DEPENDENCIES = [
+    # Keep sorted.
+    "//rs/certification/test-utils",
+]
+
+rust_library(
+    name = "read_state_response_parser",
+    srcs = glob(["src/**"]),
```

nit: In case someone uses an editor that creates temp files in the same directory or something similar, having `bazel` only try to compile `*.rs` files avoid some potential issues.
```suggestion
    srcs = glob(["src/**/*.rs"]),
```